async-jms/solutions/async-jms/src/main/java/com/redhat/training/messaging/MessagingConnectionFactory.java

```diff
@@ -0,0 +1,30 @@
+package com.redhat.training.messaging;
+
+import org.springframework.stereotype.Component;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
+
+import org.apache.camel.component.jms.JmsComponent;
+import javax.jms.JMSException;
+
+
+@Configuration
+public class MessagingConnectionFactory {
+
+	// TODO: Add the connectionFactory Bean
+	@Bean
+	public JmsComponent jmsComponent() throws JMSException {
```

Comment:
Just a very small note here so maybe we can refactor this if we have time later:

You can set these kinda components via spring boot auto-config. So this means these can be set in a application.properties file. For example `camel.component.jms.configuration.connection-factory`

https://camel.apache.org/components/2.x/jms-component.html#_spring_boot_auto_configuration

ps. I was just passing by because I am checking out stuff about what we teach in this chapter, so that I can find out proper questions for the quiz. I did not have a review intention:)